OpenRA.Mods.RA/Traits/Disguise.cs

```diff
@@ -48,10 +48,16 @@ public Player Owner
 		{
 			get
 			{
-				if (disguise.Disguised)
-					return self.Owner == self.World.LocalPlayer ? self.Owner : disguise.AsPlayer;
+				if (!disguise.Disguised)
+					return self.Owner;
 
-				return self.Owner;
+				Player p = null;
+				if (self.World.LocalPlayer != null && self.World.LocalPlayer.WinState == WinState.Undefined)
```

Comment:
@obrakmann this would put the case of the player with the disable shroud cheat into `<stuff for allies>`, right? I'd assume that using that cheat should allow me to see disguised spies even for enemies.